src/documentation-app.js

```diff
@@ -36,7 +36,7 @@ class DocumentationApp extends ReduxMixin(PolymerElement) {
         --app-primary-color: #3c3c3b;;
         --app-secondary-color: white;
       
-        --button-active-background-color: #1b4385;
+        --button-active-background-color: #a175ea;
```

Comment:
<img width="1068" alt="Zrzut ekranu 2024-05-25 o 23 05 34" src="https://github.com/FACT-Finder-Web-Components/documentation/assets/37622914/d4a911a4-b41c-45ef-b603-554a2910aaf0">
This color was not specified in the presentation, so I just used color picker to extract it